lib/convection/model/template/resource/aws_elbv2_target_group.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../resource'
+
+module Convection
+  module Model
+    class Template
+      class Resource
+        ##
+        # AWS::ElasticLoadBalancingV2::TargetGroup
+        ##
+        class ELBV2TargetGroup < Resource
+          include Model::Mixin::Taggable
+
+          type 'AWS::ElasticLoadBalancingV2::TargetGroup', :elbv2_target_group
+          property :health_check_interval_seconds, 'HealthCheckIntervalSeconds'
+          property :health_check_path, 'HealthCheckPath'
+          property :health_check_port, 'HealthCheckPort'
+          property :health_check_protocol, 'HealthCheckProtocol'
+          property :health_check_timeout_seconds, 'HealthCheckTimeoutSeconds'
+          property :healthy_threshold_count, 'HealthyThresholdCount'
+          property :matcher, 'Matcher', :type => :hash
+          property :name, 'Name'
+          property :port, 'Port'
+          property :protocol, 'Protocol'
+          property :target_group_attributes, 'TargetGroupAttributes', :type => :list
+          property :targets, 'Targets', :type => :list
+          alias_method :target_descriptions, :targets
```

Comment:
See:  http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-targetgroup.html

The json for the TargetGroup type says "Targets", but the actual object type is the "TargetDescription":
http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticloadbalancingv2-targetgroup-targetdescription.html